src/App.tsx

```diff
@@ -14,11 +15,46 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+interface State {
+  goods: string[],
+  isStarted: boolean,
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    goods: goodsFromServer,
+    isStarted: false,
+  };
+
+  start = () => {
+    this.setState((prevState) => ({
+      isStarted: !prevState.isStarted,
+    }));
+  };
+
+  render() {
+    const { isStarted, goods } = this.state;
+
+    return (
+      <div className="App">
+        <h1 className="App__title">Goods</h1>
+        {
+          isStarted && (
+            <GoodsList goods={goods} />
```

Comment:
`GoodsList` component should only render list of goods.
All controls and buttons should be in the App.